src/io/flutter/project/FlutterProjectOpenProcessor.java

```diff
@@ -92,6 +81,16 @@ private void doPostOpenProcessing(@Nullable Project project) {
       return;
     }
 
+    if ( FlutterSdk.getFlutterSdk(project) == null) {
+      final FlutterSdk incomplete = FlutterSdk.getIncomplete(project);
+      if (incomplete != null) {
+        boolean ok = incomplete.syncShowingProgress(project);
```

Comment:
I intended this to be modal, but it's not. While we're syncing the flutter SDK, the project window opens and the analysis server starts running. As a result, the analysis server shows a bunch of errors due to missing files. Syncing the analysis server fixes it.